src/components/studies/scheduler/SchedulableSingleSessionContainer.test.tsx

```diff
@@ -555,80 +565,759 @@ describe('SchedulableSingleSessionContainer', () => {
       expect(addButton).toBeDisabled()
     })
   })
-})
 
-test.skip('renders and changes reminder notification type', async () => {
```

Comment:
The diff is a bit misleading here -- the previous notification tests were moved into the session notification subsection and are still at the bottom of this set of tests